process/datadog_checks/process/data/conf.yaml.example

```
@@ -69,11 +69,11 @@ instances:
     # ignore_denied_access: true
 
     ## @param thresholds - object - optional
-    ## The threshold paramter is composed of two ranges: critical and warning
+    ## The threshold parameter is composed of two ranges: critical and warning
     ##   * warning: (optional) List of two values: If the number of processes found is below the first value or
-    ##             above the second one, the process check will return WARNING.
+    ##             above the second one, the process check returns WARNING. Set to .inf or -.inf to disable a boundary.
```

(nit) `disable` seems not the best word for this case.

```suggestion
    ##             above the second one, the process check returns WARNING. `-.inf` and `.inf` are valid values for lower and upper bounds.
```

CC @l0k0ms maybe you have a better wording for this.

---

```To make an semi-unbounded interval, use `.inf`: for the upper bound or `-.inf`: for the lower bound ```

---

Changed this line, and removed the `-.inf` part due to the comment above.